### PR TITLE
popup window 

### PR DESCRIPTION
**Pull request by zakiyousufi**
I Created the popup window and added these changes 

- update `index.html` file 
- update `index.js` file 
- update `style.css` file 
- no linters errors
_________________________________________________________________________________
this project was pair programming because my partner didn't respond the student success told me that I should do it solo
hhhhhhh

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -128,11 +129,52 @@ <h2 class="heading2 multy-task">Multi-Post Stories</h2>
...
+                            class="animate-btn project-desktop-button button1 button1-text popup_btn">See
+                            Live <img class="see_live_icon" alt="see_live_image"
+                                src="css/Icons/see_live_Icon.png"></button>
+                        <button type="submit"
+                            class="animate-btn project-desktop-button button1 button1-text popup_btn">See
+                            Source <img class="see_live_icon" alt="see_live_image" src="css/Icons/git_hub.png"></button>
+                    </form>
+                </div>
+            </div>
+        </div>
```
Comment (muneebulrehman) [inline index.html]: Your popup is not working properly, and popup is only implemented for one project. However, all the project cards should have a popup. Also the button for last project card is not working.

File: index.html [hunk 2]
```diff
@@ -128,11 +129,52 @@ <h2 class="heading2 multy-task">Multi-Post Stories</h2>
...
         <div class="projects-section2">
```
Comment (muneebulrehman) [inline index.html]: All the cards should be generated using javascript and not manually in index.html. Kindly implement the cards dynamically with JS.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @zakiyousufi  

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

- [ ] There is no title and description for your PR. Consider adding the same to PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -128,127 +130,62 @@ <h2 class="heading2 multy-task">Multi-Post Stories</h2>
                     </li>
                 </ul>
                 <form>
-                    <button type="submit" class="animate-btn project-desktop-button button1 button1-text">See
+                    <button type="submit" class="animate-btn project-desktop-button button1 button1-text btn-active">See
```
Comment (sja-thedude) [inline index.html]: - [ ] As suggested previously, kindly display the popup for all cards. Currently, the popup is not displayed for Multi post stories as shown below. Fix it.
- [ ] As suggested previously to make all cards dynamically from js. Kindly create Multi post stories dynamically from js and remove it from HTML.

https://user-images.githubusercontent.com/70269507/174431567-4b5986cf-0e41-459f-885f-45986630fb90.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @zakiyousufi, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Great job so far

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koachcode — APPROVED** [review]
Hi @zakiyousufi , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**